website/src/_posts/2022-12-3.3.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "🎄 Uppy 3.1—3.3: Improved AWS S3 Multipart, Single File Mode" 
+date: 2022-12-23
+author: 
+  - aduh95
+  - arturi
+  - murderlon
+  - mifi
+image: "https://uppy.io/images/blog/3.1-3.3/single-file-mode.jpg"
+published: true
+---
+
+Three minor releases ago we’ve released Uppy 3.0. It’s time to give you an update on what we’ve been working on all this time.
+
+TL;DR: AWS S3 Multipart stability improvements, Single File Mode, more tests and bugfixes, new locales.
```

Comment:
```suggestion
In short, AWS S3 Multipart has become more stable on slow connections, Single File Mode for the Dashboard has arrived, and many other things.
```

Reply:
If we list two things, “and many other things” doesn't sound that good to me.